The build system creates some symlinks that are useful when developing
but get in the way when we install for more than one python. Remove
these.

Not a patch for upstream, as it makes development slightly more
complicated.

Patch by Arfrever.

--- setup.py
+++ setup.py
@@ -57,12 +57,3 @@
         packages=['pyalsa'],
         scripts=[]
 )
-
-uname = os.uname()
-a = 'build/lib.%s-%s-%s' % (uname[0].lower(), uname[4], sys.version[:3])
-for f in ['alsacard.so', 'alsacontrol.so', 'alsahcontrol.so', 'alsamixer.so', 'alsaseq.so']:
-        if not os.path.exists('pyalsa/%s' % f):
-                a = '../build/lib.%s-%s-%s/pyalsa/%s' % (uname[0].lower(),
-                        uname[4], sys.version[:3], f)
-                print a, f
-                os.symlink(a, 'pyalsa/%s' % f)

